src/game/logic/ai/turretai.h

```diff
@@ -277,7 +279,7 @@ class TurretAIHoldTurretState : public TurretState
 public:
     TurretAIHoldTurretState(TurretStateMachine *machine);
 
-    virtual ~TurretAIHoldTurretState() override;
+    virtual ~TurretAIHoldTurretState() override {}
```

Comment:
the member in this class likely will be m_nextRecenterTime, tho could be m_nextIdleScan as well
comment in ini is `RecenterTime       = 5000     ; how long to wait during idle before recentering`

Reply:
fixed.